examples/ddpg_series_omega_control.py

```diff
@@ -11,54 +11,93 @@
 from rl.memory import SequentialMemory
 from rl.random import OrnsteinUhlenbeckProcess
 from gym.wrappers import FlattenObservation
-import sys, os
+import sys
+import os
 sys.path.append(os.path.abspath(os.path.join('..')))
 import gym_electric_motor as gem
 from gym_electric_motor.reference_generators import WienerProcessReferenceGenerator
 from gym_electric_motor.visualization import MotorDashboard
 
+# This example shows how we can use GEM to train a reinforcement learning agent to control the motor speed
```

Comment:
Additional information in this abstract of the example regarding the motor type and that we are operating in a cont. state space would be nice